src/include/cpu/power/istep_13.h

```diff
@@ -0,0 +1,5 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <device/dram/ddr4.h>
+
+void istep_13_2(dimm_attr dimms[CONFIG_DIMM_MAX]);
```

Comment:
```suggestion
void istep_13_2(dimm_attr *dimms);
```

Wouldn't be better? Otherwise DIMM_MAX would change the function declrataion

Reply:
That was the point, the compiler should warn if old object files are trying to be linked with newer ones. This still shouldn't happen with properly set Makefile/Kconfig. Function gets the pointer as an argument anyway, this doesn't change. What changes is that it enables some additional compile-time boundary checks and possibly optimizations. Number of DIMM slots doesn't change from one boot to another so I don't see why we need to support unknown number of elements in the array.